src/RA_RichPresence.cpp

```diff
@@ -193,7 +193,7 @@ static bool GetLine(ra::services::TextReader& pReader, std::string& sLine)
         if (index != std::string::npos)
         {
             // if a comment marker was found, remove it and any trailing whitespace
-            while (index > 0 && isspace(sLine[index - 1]))
+            while (index > 0 && isspace(gsl::narrow<int>(sLine.at(index - 1))))
```

Comment:
`sLine` is a string, so an element of `sLine` is a character. `narrow` seems like an inaccurate conversion to get to `int`.

Reply:
At returns a char, would it be better to cast it to unsigned char instead?

Reply:
I'll look up a better way to do this. Narrow is basically a static cast that that throws if casting changes the value